Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rust:bookworm as builder
+
+WORKDIR /usr/src/sentrum
+COPY . .
+
+RUN cargo build --release
+
+FROM debian:bookworm-slim
+
+# Upgrade all packages and install dependencies
+RUN apt-get update \
+    && apt-get upgrade -y
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends libssl-dev \
+    && apt clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY --from=builder /usr/src/sentrum/target/release/sentrum /usr/local/bin/sentrum
+
+COPY sentrum.sample.toml sentrum.toml
+
+CMD ["sentrum"]
```

Comment:
nit: seems like your are missing some newlines at the end of some files.

Reply:
Ah, yes, will fix!

Reply:
Fixed in https://github.com/sommerfelddev/sentrum/pull/4/commits/a57c12d273e14334c2a91cbc8165163ea859540e